2_feature_importance/2_feature_importance.ipynb

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
_Commented on notebook `2_feature_importance/2_feature_importance.ipynb` Cell 11 Line 15_
```python

for input_feature in X_test.columns:
    # Mean premutation importance:
    # Takes the input feature mean to determine performance
    X_test_permuted = X_test.copy()
    X_test_permuted[input_feature] = X_test_permuted[input_feature].mean()
    
    permuation = {"date": X_test.index,
```
Minor detail here, but you wrote "permuation" instead of permutation *(PTSD intensifies)*.

_Respond and view the context [here](https://gitnotebooks.com/oriol-pomarol/codegeo_workshops/pull/1)._

Reply:
Replaced all instances